tests/python/pytorch/graphbolt/test_sampling.py

```diff
@@ -0,0 +1,164 @@
+import dgl
+import dgl.graphbolt as gb
+import dgl.sparse as dglsp
+import torch
+
+
+def test_sampling():
+    def setup_seed(seed):
```

Comment:
Move this util function to https://github.com/dmlc/dgl/blob/master/tests/python/pytorch/graphbolt/gb_test_utils.py